runtime-modules/proposals/codex/src/lib.rs

```diff
@@ -50,7 +50,7 @@
     deny(clippy::match_on_vec_items),
     deny(clippy::unreachable)
 )]
-
+#![feature(int_roundings)]
```

Comment:
This is only required for tests, so how about:
```suggestion
#![cfg_attr(test, feature(int_roundings))]
```